express.js

```diff
@@ -10,6 +10,7 @@ let expressValidator = require('express-validator');
 let bodyParser = require('body-parser');
 let passport = require('passport');
 let OAuth2Strategy = require('passport-oauth2');
+let schedule = require('node-schedule');
```

Comment:
Please do not add new dependencies to the website unless you need a new feature. The schedule job change you've added does exactly the same as an interval, except we now have more dependencies.